src/Phinx/Db/Adapter/PostgresAdapter.php

```diff
@@ -1297,7 +1297,7 @@ public function createSchemaTable()
     public function createSchema($schemaName = 'public')
     {
         // from postgres 9.3 we can use "CREATE SCHEMA IF NOT EXISTS schema_name"
-        $sql = sprintf('CREATE SCHEMA %s', $this->quoteSchemaName($schemaName));
+        $sql = sprintf('CREATE SCHEMA IF NOT EXISTS %s', $this->quoteSchemaName($schemaName));
```

Comment:
What about the comment and <9.3?

Reply:
postgresql 9.3 was end of life back in 2018, and I do not think it should be a target for support of phinx.